### DISCUSSION TITLE
What you think about NVD database being part of initial download?

### DISCUSSION THREAD

**Discussion by austinsonger** [Secure-Compliance-Solutions-LLC/GVM-Docker#133] [General]
Right now you have to build the NVD database from scratch when creating the docker container. What would you think about us creating another repo that builds the NVD database that during the build it downloads from that repo?

**Comment by Dexus** [comment]
A problem that I see is that the first initiation takes around 45 minutes before the complete db are ready to use. I see in the logs of gvmd that it build the DB and sync all the files. In my tries I saw the config files are first at the end are loaded, and then also available in the web UI.
